tests/test_resources/test_parse_yaml.yaml

```diff
@@ -0,0 +1,2 @@
+test:
+  parse: yaml
```

Comment:
For consistency, can you also add a yaml array here with some items? To test yaml syntax a bit more strictly.